### PR TITLE
Add TimeTable UI And Logic

### PR DESCRIPTION
**Pull request by yeggasd**
#142

### CODE REVIEW

File: src/main/java/seedu/address/ui/TimeTablePanel.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import javafx.fxml.FXML;
+import javafx.scene.control.TableCell;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.layout.Region;
+import javafx.scene.paint.Color;
+
+//@@author yeggasd
+/**
+ * A ui for the info panal that is displayed when the vacant command is called.
```
Comment (jingyinno) [inline src/main/java/seedu/address/ui/TimeTablePanel.java]: Here too

File: src/main/java/seedu/address/ui/TimeTablePanel.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            columns.get(i).setCellValueFactory(cellData -> new SimpleStringProperty(cellData.getValue().get(j)));
+            if (j != 0) {
+                columns.get(i).setMinWidth(100);
+                columns.get(i).setMaxWidth(300);
+            }
+        }
+    }
+
+    /**
+     * Sets the command box style to indicate a vacant or occupied room.
```
Comment (jingyinno) [inline src/main/java/seedu/address/ui/TimeTablePanel.java]: Here too

File: src/main/java/seedu/address/logic/commands/TimeTableCommand.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+import seedu.address.commons.core.EventsCenter;
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.events.ui.TimeTableEvent;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.Person;
+
+//@@author yeggasd
+/**
+ * Retrieves all vacant rooms in a given building
```
Comment (jingyinno) [inline src/main/java/seedu/address/logic/commands/TimeTableCommand.java]: Rmb to change the comments hahaha

File: src/main/java/seedu/address/ui/TimeTablePanel.java (hunk 1 above)
Comment (yeggasd) [reply src/main/java/seedu/address/ui/TimeTablePanel.java]: panal

**Comment by jingyinno** [comment]
Thanks for collate

**Pull request merged**